.travis.yml

```diff
@@ -27,6 +27,7 @@ matrix:
           packages:
           - gnutls-dev
 #    - env: variant=test_debug tests=1 crypto=wolfssl toolset=gcc sanitizer=on
+    - env: variant=test_debug tests=1 crypto=openssl webtorrent=on toolset=gcc sanitizer=on
```

Comment:
one thing that has changed in master is the names of the "variant"s. change "test_debug" to just "debug"

Reply:
OK!